src/pages/partner.mdx

```diff
@@ -42,35 +42,38 @@ recommend them to the userbase, and manufacturers can suppcomort the developers
 necessary hardware. The userbase will then benefit from great hardware, and the future of the
 project will be secure.
 
-<b className="flex flex-col items-center">
+<br></br>
+<br></br>
+<p className="flex flex-col items-center">
+  <img src="/img/sponsors/bf_partner_dark.svg" className="no-effect md:w-1/4"></img>
+</p>
+<br></br>
 
-<img src="/img/BF_Partner_Transparent.png" className="no-effect md:w-1/3"></img>
+<p className="flex flex-col items-center">
 
-We are happy to announce the 'Betaflight partner' manufacturers for release 4.4 are:
+#### The 'Betaflight partner' manufacturers for release 4.4 are:
```

Comment:
Should this not be TitleCase?

Reply:
I just used the formatting of the header but I guess I can remove it and format the text using HTML and CSS. Let me see...